### PR TITLE
Update sixlabors 3.0.1

### PR DESCRIPTION
**Pull request by AlexanderSeel**
# Pull Request

### General

- [x ] I have added tests for my changes.
- [x ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x ] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-blazor/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component

### CODE REVIEW

File: src/Baaijte.Optimizely.ImageSharp.Web/Baaijte.Optimizely.ImageSharp.Web.csproj
Comment (vnbaaij) [inline src/Baaijte.Optimizely.ImageSharp.Web/Baaijte.Optimizely.ImageSharp.Web.csproj:1]: Happy to name you as a contributor, but not going to mark you as author and repo

File: src/Baaijte.Optimizely.ImageSharp.Web/Baaijte.Optimizely.ImageSharp.Web.nuspec
Comment (vnbaaij) [inline src/Baaijte.Optimizely.ImageSharp.Web/Baaijte.Optimizely.ImageSharp.Web.nuspec:1]: Same remarks, don't change author/owner/repo links

**Review by vnbaaij — CHANGES_REQUESTED** [review]

File: src/Baaijte.Optimizely.ImageSharp.Web/Baaijte.Optimizely.ImageSharp.Web.csproj
Comment (AlexanderSeel) [reply src/Baaijte.Optimizely.ImageSharp.Web/Baaijte.Optimizely.ImageSharp.Web.csproj:1]: Oh sorry this was not intentional

Comment (AlexanderSeel) [reply src/Baaijte.Optimizely.ImageSharp.Web/Baaijte.Optimizely.ImageSharp.Web.csproj:1]: Hi, can you give me a hint, I just don't want to be the owner. Which checkmark I missed. Kind gerads

**Review by AlexanderSeel — COMMENTED** [review]
changed project url, author and owner

**Review by vnbaaij — APPROVED** [review]

**Pull request merged**